msig_contract_deployment/msig_erc20_account_creation.md

```diff
@@ -0,0 +1,33 @@
+<h1>Multi-sig account creation of evm accounts for EOS-ERC20 token bridge </h1>
+
+<h2>Overview</h2>
+EOS-EVM will soon support trustless token bridge between tokens on native EOS (for example: USDT on EOS) and ERC-20 compatible tokens on EOS-EVM. (contract implementaion can be found in https://github.com/eosnetworkfoundation/evm-bridge-contracts). Because of such, we need the multi-sig from 21 block producers to create the following accounts on testnet & mainnet:
```

Comment:
Typo: implementaion

Reply:
Complete the sentence by saying we need to create the account `eosio.erc2o` and `eosio.evmin` within this paragraph.

Then the reader knows what to expect when for each of the headings below when you talk about how to setup the permissions for each account.